tests/ga/test_update.py

```diff
@@ -2164,19 +2163,16 @@ def _mock_update_handler(iterations=1, **kwargs):
                 
                 update_handler.protocol_util.get_protocol = Mock(return_value=protocol)
 
-                original_report_status = update_handler._report_status
-                update_handler._report_status = Mock(side_effect=original_report_status)
-
                 is_running_patch.start()
                 yield update_handler
         finally:
             is_running_patch.stop()
 
     @staticmethod
-    def _goal_state_handler(url, **kwargs): # pylint: disable=unused-argument
+    def _fail_goal_state_fetch(url, **_): # pylint: disable=unused-argument
```

Comment:
you dont need the pylint warning suppression if using `**_` variable naming convention, please remove it here